docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '2.4'
 services:
   db:
     image: postgres
```

Comment:
need to be updated to use amd64 variant `amd64/postgres`
refer: https://hub.docker.com/r/amd64/postgres